backend/config/development.yaml

```diff
@@ -8,4 +8,8 @@ development:
     - 'http://localhost:8000'
     - 'http://127.0.0.1:8000'
     - 'http://localhost:3000'
-    - 'http://127.0.0.1:3000'
\ No newline at end of file
+    - 'http://127.0.0.1:3000'
+  SOCIAL_AUTH_GOOGLE_OAUTH2_KEY: '172952687211-u1a8hlv7eeprjmfnjfgbc4jfj5phmu3g.apps.googleusercontent.com'
+  SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET: 'x5tKtbh2SPRJ2AA3EuyiDuE8'
```

Comment:
Can we define it in more structured way? Like `SOCIAL_AUTH` -> `GOOGLE` -> `key/secret`?